module.nix

```diff
@@ -280,6 +280,26 @@ in
         services = {
           authentik.settings = {
             blueprints_dir = mkDefault "${cfg.authentikComponents.staticWorkdirDeps}/blueprints";
+            listen = {
+              # From Authentik 2025.8.4 and onwards the "listen" options take the following form
+              debug = mkDefault "127.0.0.1:9900";
+              debug_py = mkDefault "127.0.0.1:9901";
+              http = mkDefault "127.0.0.1:9000";
+              https = mkDefault "127.0.0.1:9443";
+              ldap = mkDefault "0.0.0.0:3389";
+              ldaps = mkDefault "0.0.0.0:6636";
+              radius = mkDefault "0.0.0.0:1812";
+              metrics = mkDefault "0.0.0.0:9300";
+              # From Authentik before 2025.8.4 settings were prefixed with 'listen_' as follows
```

Comment:
First of all, this is incorrect. This was a bug that only affected the worker: https://github.com/goauthentik/authentik/issues/16851

Also, we never supported other authentik versions than the one packaged together with the module.

Reply:
changed the PR.